packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjectsContainer.js

```diff
@@ -1,7 +1,8 @@
-import useSWR from 'swr'
 import { shape, string } from 'prop-types'
+import { panoptes } from '@zooniverse/panoptes-js'
+import useSWR from 'swr'
 
-import fetchRecentSubjects from './fetchRecentSubjects.js'
+import { usePanoptesProjects } from '@hooks'
```

Comment:
Rather that use a custom fetching function, this container can actually use the shared `usePanoptesProjects` hook to get project data for multiple projects at the same time.